redbot/core/cli.py

```diff
@@ -138,6 +138,16 @@ def parse_cli_flags(args):
         action="store_true",
         help="Enables the built-in RPC server. Please read the docs prior to enabling this!",
     )
+    parser.add_argument("--token", type=str, help="Run Red with the given token.")
+    parser.add_argument(
+        "--no-instance",
+        action="store_true",
+        help=(
+            "Run Red without any existing instance. "
+            "The data will be saved under a temporary folder "
+            "and deleted on next restart."
```

Comment:
"deleted on next restart" could be interpreted as the next bot restart. I would specify "system restart" to be clearer.